systemtest/src/test/java/io/strimzi/systemtest/operators/topic/TopicST.java

```diff
@@ -485,8 +482,6 @@ void testKafkaTopicDifferentStatesInBTOMode() throws InterruptedException {
      *  - kafkatopic-not-ready
      */
     @IsolatedTest
-    @KRaftWithoutUTONotSupported
-    @BTONotSupported
```

Comment:
There is also `testKafkaTopicDifferentStatesInBTOMode` which can be removed completely. Or I can do it in different PR. It's up to you.